src/win32_window.c

```diff
@@ -31,925 +31,989 @@
 #include <stdlib.h>
 #include <malloc.h>
 #include <string.h>
+#include <WinUser.h>
 #include <windowsx.h>
 #include <shellapi.h>
+#include <stdio.h>
 
 #define _GLFW_KEY_INVALID -2
 
 // Returns the window style for the specified window
 //
 static DWORD getWindowStyle(const _GLFWwindow* window)
 {
-    DWORD style = WS_CLIPSIBLINGS | WS_CLIPCHILDREN;
+	DWORD style = WS_CLIPSIBLINGS | WS_CLIPCHILDREN;
```

Comment:
The diff for this file is completely unusable because your editor has replaced the indentation of every line. GLFW uses an indentation of four spaces, no tabs.